Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -145,6 +171,37 @@ private PerformanceInfo ReadPerformanceInfo()
             return result;
         }
 
+        private NetworkUsage ReadNetworkUsageInfo()
+        {
+            var result = new NetworkUsage();
+
+            try
+            {
+                var totalReceivedBytes = 0L;
+                var totalSentBytes = 0L;
+
+                foreach (var line in networkUsageReader.ReadLines().Skip(2))
```

Comment:
1. `Skip (2)` seems arbitrary without a link to the file format description in the Linux kernel docs.

Reply:
2. We wouldn't want to measure anything about the loopback interface (`lo`), right?

Reply:
3. Please investigate how this solution interacts with Docker (relevant for future Houston hosts monitoring):
- Does it create virtual network interfaces when using bridge networking? 
- If it does, should they be included in this sum? Ideally, we don't want to count anything twice, and we only want to measure traffic going in or out the physical wire.

Reply:
> Please investigate how this solution interacts with Docker (relevant for future Houston hosts monitoring):
> 
> * Does it create virtual network interfaces when using bridge networking?
> * If it does, should they be included in this sum? Ideally, we don't want to count anything twice, and we only want to measure traffic going in or out the physical wire.

According to [this doc](https://docs.docker.com/config/containers/container-networking/), container has only one network interface.

But it seems that by default docker has 3 network interfaces [(it can be seen here)](https://docs.docker.com/network/none/). Tunnel interfaces are down by default, so they always return 0 values. 

It means, that docker shouldn't be a problem, but I'm not sure what these down interfaces were made for.

Also, there is a nice picture explaining different types of docker networks.
![image](https://user-images.githubusercontent.com/34509541/90121569-93466e00-dd75-11ea-9fc0-b7a82854d475.png)